service/controller/azure/v5/key/key.go

```diff
@@ -5,14 +5,6 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
-func ClusterID(azureClusterConfig v1alpha1.AzureClusterConfig) string {
-	return azureClusterConfig.Spec.Guest.ID
-}
-
-func ClusterOrganization(azureClusterConfig v1alpha1.AzureClusterConfig) string {
-	return azureClusterConfig.Spec.Guest.Owner
-}
```

Comment:
I removed these because I messed up here. I think we should use the shared key package when possible.